<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-EBLR0J5" failures="1" tests="8" name="CrudOperation" time="6.195" errors="0" timestamp="2024-10-24T13:13:27 IST">
  <testcase classname="test.Postmethod" name="PostTest" time="2.43"/>
  <testcase classname="test.TestExample2" name="GetSingleUser" time="0.366"/>
  <testcase classname="test.TestExample2" name="StaticRestAssured" time="0.875"/>
  <testcase classname="test.DeleteRequest" name="DeleteTest" time="0.852"/>
  <testcase classname="test.PutRequestMethod" name="PutTest" time="0.57"/>
  <testcase classname="test.PatchRequest" name="PatchTest" time="0.652"/>
  <testcase classname="test.TestExample1" name="test1" time="0.307"/>
  <testcase classname="test.TestExample1" name="test2" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.TestExample1.test2(TestExample1.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- CrudOperation -->
